beekeeper-api/src/main/java/com/expediagroup/beekeeper/api/BeekeeperController.java

```diff
@@ -15,9 +15,23 @@
  */
 package com.expediagroup.beekeeper.api;
 
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import com.expediagroup.beekeeper.core.model.HousekeepingMetadata;
+
 @RestController
 public class BeekeeperController {
-  
+
+  private HousekeepingEntityService housekeepingEntityService;
+
+  @RequestMapping("/tables")
```

Comment:
Can you specify the type of mapping explicitly or change the annotation to `@GetMapping`? Would also be nice to explicitly state the arguments you're setting so its easy to know